modal/mount.py

```diff
@@ -406,9 +424,10 @@ async def _get_files(entries: List[_MountEntry]) -> AsyncGenerator[FileUploadSpe
 
             futs = []
             for local_filename, remote_filename in all_files:
+                logger.debug(f"Mounting {local_filename} as {remote_filename}")
```

Comment:
Could this potentially make the debug logs overwhelmingly verbose for users mounting large dependencies?

Reply:
They already have every MountPutFile RPC call logged, so at most this is a 2x increase, but the "metadata" is way more useful than the RPC call here